core/vibes/soul/primitives/product-card/index.tsx

```diff
@@ -123,7 +126,7 @@ export function ProductCard({
           <div className="flex-1 text-sm @[16rem]:text-base">
             <span
               className={clsx(
-                'block font-semibold',
+                'line-clamp-2 font-semibold',
```

Comment:
Can we include changes like these in a separate PR for review?

Reply:
Why can't we get it reviewed in this PR? 🤔